src/main/java/com/Acrobot/ChestShop/ChestShop.java

```diff
@@ -536,6 +539,13 @@ public static DrilldownPie createStaticDrilldownStat(String statId, String value
         return new DrilldownPie(statId, () -> map);
     }
 
+    public static DrilldownPie createStaticDrilldownStat(String statId, Callable<EconomyAdapter.ProviderInfo> callableProviderInfo) {
+        return new DrilldownPie(statId, () -> {
+            EconomyAdapter.ProviderInfo providerInfo = callableProviderInfo.call();
+            return ImmutableMap.of(providerInfo.getName(), ImmutableMap.of(providerInfo.getVersion(), 1));
```

Comment:
This will throw an NPE when no provide is set. I realize that this shouldn't be happening as the plugin disables on load but you made getProviderInfo Nullable so it should be checked here.